### DISCUSSION TITLE
Welcome to Flutter-UI Discussions!

### DISCUSSION THREAD

**Discussion by ankitoscar** [Project-Easter/Flutter-UI#90] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by keyurgit45** [comment]
Issue #112: I think that the blue button with the logo of location is a floating action button not a bottom navigation button @Mohitmadhav

**Reply by Mohitmadhav** [reply]
Sorry, my bad, changed it !!

**Comment by Mohitmadhav** [comment]
@keyurgit45 would you like to work on Forgot Password section? It's the last stage of auth for now.

**Reply by keyurgit45** [reply]
OK I will take that for now. you can assign me

**Reply by keyurgit45** [reply]
And flow of screen for forget password is in the figma right?

**Reply by Mohitmadhav** [reply]
I'm creating the issue within a minute, you just comment there.
You'll get all the details there.

**Reply by Mohitmadhav** [reply]
Or let it be, you can work on other Firebase issues as they are more important right now.

**Reply by keyurgit45** [reply]
Well! I can take #92. There are already Models created for user and books in lib/providers. So I need to make function that makes firestore collection from that for each uid right? @Mohitmadhav

**Reply by Mohitmadhav** [reply]
Yes, tag me there and I'll assign it to you.

**Reply by keyurgit45** [reply]
> Yes, tag me there and I'll assign it to you.
> 

Can you give more details about my task

There is already function makebook implemented for add book button

**Reply by keyurgit45** [reply]
@Mohitmadhav can you please delete a document in ownbooks collection of this uid : StfcwKnMtzXMnN8VlzSKFpYA0jH3 ?

**Reply by Mohitmadhav** [reply]
Yeah did that @keyurgit45

**Comment by keyurgit45** [comment]
@Mohitmadhav when you are going to create new issues?

**Reply by Mohitmadhav** [reply]
Will create some by tomorrow at max.

**Comment by ankitoscar** [comment]
@Mohitmadhav these issues were assigned 5 days ago but no updates have been provided since then:
1. Issue number #79 assigned to @merelj
2. Issue number #95 assigned to @Phantom0001
3. Issue number #99 assigned to @Mastersam07
4. Issue number #109 assigned to @CoderNamedHendrick

Please give some reply otherwise you will be unassigned from your respective issues and they will be assigned to someone else.

**Reply by boredcity** [reply]
Hi there.
Been swamped all week unfortunately 😕

I will try to go back to the task during the weekend, but if someone else is excited to give it a go I'll gladly give it up for them 😄

**Reply by Mohitmadhav** [reply]
Also, @merelj notify us if you won't be able to do

**Reply by Mohitmadhav** [reply]
@Mastersam07 @CoderNamedHendrick the issues will be unassigned by tomorrow if you are not able to catch up.

**Reply by CoderNamedHendrick** [reply]
Hello.
 I'm sorry, I have been busy all week, as @merelj said, I'll give it a go this weekend.

**Comment by ankitoscar** [comment]
@Mohitmadhav I have unassigned issue numbers #95 and #99  assigned to @Phantom0001 and @Mastersam07 respectively. Anyone who wants to take those issues, comment there and @Mohitmadhav will assign it to you.

**Reply by boredcity** [reply]
please unassign #79 also

**Comment by keyurgit45** [comment]
@Mohitmadhav I changed some code now it is working on my device. I have pushed the code too.

https://user-images.githubusercontent.com/75253653/137350979-61dc8fd0-567d-491a-8c90-b289f57f1f66.mp4

.

**Reply by Mohitmadhav** [reply]
Ok, that seems fine...
Just pull the latest changes from upstream main

**Reply by keyurgit45** [reply]
Done!

**Comment by keyurgit45** [comment]
https://user-images.githubusercontent.com/75253653/139215908-aa17e905-03b0-4fd9-be85-4e658c714698.mp4

@Mohitmadhav

**Reply by Mohitmadhav** [reply]
Ok @keyurgit45 I'll check...you create the PR
